### ISSUE TITLE
Change e-mail adress of demo accounts

### ISSUE THREAD

**Issue by DonnaDuiker** [cherrytwist/Server#4779]
## Description

As a member of the product team
I want to have control over the demo accounts
So that I can make changes to the account and it's current contributions independently of Donna (the current owner of the accounts)

## Acceptance criteria

- [ ] Start with University teacher account so Donna can make sure everything is right before changing the others
- [ ] After Donna checked everything is fine, continue with the others 

For Donna:
- [ ] Still in all (Sub)Spaces
- [ ] Contributions still exist
- [ ] Comments still excist

List of demo accounts:

<img width="1103" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM1FmQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--65cf037807663daaf1ef029e518b65b7b7ffd688/image.png" alt="image.png" />

[TransferAccounts.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM29mQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a14916112203d24253a3a93b06862cda4e22ad91/TransferAccounts.xlsx)

## Additional Context
For Donna:
If we start with the University teacher account:
Profile page: https://alkem.io/user/donnatest-duiker
In space: https://alkem.io/universitydemo/dashboard
And Subspace: https://alkem.io/universitydemo/challenges/digitaltransformationst
Created tool and comment: https://alkem.io/universitydemo/challenges/digitaltransformationst/collaboration/tipsfromteacher

## Areas that will be affected 
 
_To be added during the refinement_

**Comment by SimoneZaza** [comment]
@valentinyanakiev can you help @DonnaDuiker with this (after the release is fine, no rush)

**Comment by SimoneZaza** [comment]
@DonnaDuiker is this still open?

**Comment by DonnaDuiker** [comment]
@SimoneZaza  No, it is finished, must have overlooked this, think I forgot to assign myself.
Is the protocol here to close it, or mark as finished somehow?

**Comment by SimoneZaza** [comment]
No worries, I'll close it, thanks!
